src/V3Width.cpp

```diff
@@ -5387,6 +5387,15 @@ class WidthVisitor final : public VNVisitor {
         userIterateAndNext(nodep->passsp(), nullptr);
         userIterateAndNext(nodep->failsp(), nullptr);
     }
+    void visit(AstAssertCtl* nodep) override {
+        if (nodep->assertTypesp()
+            && nodep->assertTypesp()->exists([](AstNodeBiop*) { return true; })) {
```

Comment:
Why specifically test for Biops e.g. could there be a function call here instead which presumably would break?  Should this instead look for _logically-not_ _something-that-is-legal_?

Reply:
Previously it failed in this phase when any `AstNodeBiop` was under `AstAssertCtl`. However, I recently found out that adding support for Biop would be easier than those unsupported checks. I'd pushed a commit with added support: https://github.com/verilator/verilator/pull/5236/commits/7f7f61a669fca9cea75417afbeca5cc06816f590.